src/main/java/fi/okm/jod/yksilo/repository/YksiloRepository.java

```diff
@@ -28,4 +31,8 @@ public interface YksiloRepository extends JpaRepository<Yksilo, UUID> {
 
   @Query(value = "SELECT k.uri FROM Yksilo y JOIN y.ammattiKiinnostukset k WHERE y = :yksilo")
   Set<String> findAmmattiKiinnostukset(Yksilo yksilo);
+
+  @Query(
+      value = "SELECT y FROM Yksilo y WHERE y.muokattu > :muokattuJalkeen ORDER BY y.muokattu ASC")
```

Comment:
muokattu is not guarateed to be unique, so the sort is not stable; append primary key to the ordering to make it _total_. It might also make sense to add an index on the muokattu column.